.travis.yml

```diff
@@ -1,9 +1,22 @@
-language: vim
+language: ruby
+rvm:
+  - 1.8.7 # Use stock
+  - 1.9.2 # Build latest
 
-install: |
-  sudo apt-get update
-  sudo apt-get install vim
+before_script: |
   cd ..
+  if [ $(ruby -e 'puts RUBY_VERSION') == '1.8.7' ]; then
+    sudo apt-get -y update
+    sudo apt-get -y install vim
+    sudo ln -s /usr/bin/vim /usr/local/bin/vim
+  else
+    git clone --depth 1 https://github.com/vim/vim
```

Comment:
@cirosantilli Is there a reason to given vim's conservative release strategy? Current tag is `v7-4-728`, if I do `git diff master..v7-4-728` there is no difference. AFAIK, the master will always point to latest stable release. This is the same way their hg mirror has been managed.